k8s/namespaces/monitoring/kube-prometheus-stack/patches/cftptl/cluster-00/kube-prometheus-stack.yaml

```diff
@@ -107,3 +107,19 @@ spec:
           type: camptocamp-prometheus-alertmanager-datasource
           url: http://alertmanager-01.platform.hmcts.net
           access: proxy
+        - name: sds-prometheus-dev-00
```

Comment:
hmm we probably don't want these here? they won't be able to reach SDS atm and we plan to deploy new SDS PTL clusters to match CFT